components/order-app/stores/fulfillment-store.ts

```diff
@@ -3,16 +3,16 @@ import { distanceFromTMS } from './order-utils';
 import DateStore from './date-store';
 
 class FulfillmentStore {
-  option: 'pickup' | 'delivery';
+  option: 'pickup' | 'delivery' = 'delivery';
```

Comment:
Pickup should be the default

Reply:
Ok I will make 'pickup' the default option